torchao/experimental/ops/mps/register.mm

```diff
@@ -65,6 +57,26 @@ Tensor linear_mps_kernel(
       ": expect SZ to be 3d tensor with sizes [:, ",
       N,
       ", 2]");
+}
+
+template <int nbit>
+Tensor linear_mps_kernel(
+    const Tensor& A,
+    const Tensor& B,
+    int64_t group_size,
+    const Tensor& SZ) {
+  TORCH_CHECK(
```

Comment:
Should we use TORCHAO_CHECK?

Reply:
done!

Reply:
Probably in a later PR, when I register ExecuTorch ops